anta/tests/security.py

```diff
@@ -555,12 +528,22 @@ def test(self) -> None:
 
 
 class VerifyIPv4ACL(AntaTest):
-    """Verifies the configuration of IPv4 ACLs.
+    """Verifies the IPv4 ACLs.
+
+    This test performs the following checks for each IPv4 ACL:
+
+      1. Validates that the IPv4 ACL is properly configured.
+      2. Validates that the sequence entries in the ACL are correctly ordered.
```

Comment:
Lets not change it here but given the model I feel like we also support IPv6 in this test. @carl-baillargeon